paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,20 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
 {
-    protected $default_title;
-
-    protected $default_description;
-
-    protected $default_logo;
+	/**
+	 * panther restapi responses
+	 */
+	protected const PAYPRO_API_RES_NOT_SUBSCRIBED = "Not subscribed to money transfer service";
```

Comment:
This message seems to belong to PayProApiHelper, could we define it there?

Reply:
done